src/runtime_src/core/edge/user/aie/common_layer/adf_runtime_api.cpp

```diff
@@ -1,3 +1,4 @@
+// 67d7842dbbe25473c3c32b93c0da8047785f30d78e8a024de1b57352245f9689
 /**
 * Copyright (C) 2021 Xilinx, Inc
```

Comment:
Update to 2021-2022?

Reply:
Will update it using another pull request